README.rst

```diff
@@ -144,6 +144,22 @@ argument to ``bin/crawl-foo-org``. The crawler will then only fetch and index
 that specific URL.
 
 
+Slack-Notifications
+-------------------
+
+The ``ftw.crawler`` supports Slack-Notifications. Those notifications can be used
+to monitor the crawler on possible errors while crawling.
+To enable slack-notifications for your environment, you need to do the following things:
+
+- Make sure `slacker <https://github.com/os/slacker>`_ is installed in your env.
```

Comment:
I'm not sure what you mean by that, as you have already listed `slacker` in the `setup.py`. What exactly do I have to do?